scripts/write-svg-icons.js

```diff
@@ -177,12 +177,14 @@ let makeSVG = (shapeTitle, shapeContent) => {
 ${closingTag}`;
 };
 
-let convertToCamelCase = kebabCase => {
-  // will convert core-shapes into CoreShapes
-  return kebabCase
-    .split('-')
-    .map(splitPart => splitPart.charAt(0).toUpperCase() + splitPart.slice(1))
-    .join('');
+let convertToExportedName = setName => {
+  // will convert core into CoreShapes
+  return (
+    setName
+      .split('-')
```

Comment:
I don't think we need to split anymore, just uppercase the first letter and append `Shapes`.

Reply:
We do.. because `text-edit` :)